src/MultiAgentReinforcementLearning/src/policies/multi_agent.jl

```diff
@@ -0,0 +1,56 @@
+export MultiAgentManager
+
+Base.@kwdef mutable struct MultiAgentManager{T <: AbstractPolicy} <: AbstractPolicy
+    agents::Dict{S,T} where {S<: Union{String, Symbol, Integer}}
+    cur_player::Union{String, Symbol, Integer} = first(keys(agents))
```

Comment:
and
```suggestion
    cur_player::S = first(keys(agents))
```

Reply:
Nice suggestion. I changed it. Will be present in next version.